tests/serverspec/Gemfile

```diff
@@ -4,8 +4,10 @@ source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
-# gem "rails"
+# Should all go away after net-ssh releases 6.3.0
+# Ref: https://github.com/net-ssh/net-ssh/pull/838
+gem 'net-ssh', github: 'net-ssh/net-ssh.git', ref: 'a45f54f'
 
-gem "serverspec", "~> 2.41"
+gem "serverspec"
```

Comment:
Opted to leave this unrestricted since the `Gemfile.lock` will be the strict enforcement

Reply:
Sounds like a plan.